src/Compilers/Core/CodeAnalysisTest/ObjectSerializationTests.cs

```diff
@@ -31,9 +388,6 @@ public void TestRoundTripPrimitiveArray()
             var inputUShort = new ushort[] { 1, 2, 3, 4, 5 };
             var inputString = new string[] { "h", "e", "l", "l", "o" };
```

Comment:
for array tests do we try writing all the different sub-array lengths?  I want to make sure we're exercising the Array_0, Array_1... cases.